actors/market/tests/on_miner_sectors_terminate.rs

```diff
@@ -187,11 +187,13 @@ fn terminate_valid_deals_along_with_just_expired_deal() {
         CLIENT_ADDR,
         PROVIDER_ADDR,
         &[sector_number],
+        // Deal2 isn't terminated (or cleaned up) because it expired and is waiting for settlement.
         &[id0, id1],
     );
     assert_deal_deleted(&rt, id0, &deal0, sector_number);
     assert_deal_deleted(&rt, id1, &deal1, sector_number);
-    assert_deal_deleted(&rt, id1, &deal2, sector_number);
```

Comment:
This mistake (`id1`) was introduced when synchronous deal termination was introduced. The test made a lot more sense before that, and _just_ remains relevant.